README.md

```diff
@@ -8,6 +8,8 @@ This project contains information about me and my projects It shows the work I h
 
 * CSS
 
+* Javascript
+
 ## Getting Started
 
 **To get this project up and running in your local environment;**
```

Comment:
- [ ] [OPTIONAL] In the `Getting started` section, also add the necessary code for running the project locally. For ex:-
```
git clone https://github.com/kemigabocatherine/Personal---Portfolio.git
cd Personal---Portfolio
```